user.coffee

```diff
@@ -0,0 +1,61 @@
+fs = require('fs')
+
+reservedNicks = fs.readFileSync('reserved_nicks').toString().trim().split('\n')
+nick_session_map = {}
+banned_sessions = []
+status = {}
+
+# Verify the nick of user
+verify_nick = (nick)->
+  if nick == null or nick == ""
+    return false
+
+  for reservedNick in reservedNicks
+    if nick.toLowerCase().indexOf(reservedNick) != -1
+      return false
+
+  return true
+
+# Verify the session of user
+verify_session = (session_id)->
+  if session_id == null or session_id == undefined or session_id == ""
```

Comment:
and `typeof sessionId == undefined` or `sessionId.length == 0` gives error when sessionId is set to null